klog.go

```diff
@@ -1241,6 +1241,22 @@ func (v Verbose) Infof(format string, args ...interface{}) {
 	}
 }
 
+// InfoS is equivalent to the global InfoS function, guarded by the value of v.
+// See the documentation of V for usage.
+func (v Verbose) InfoS(msg string, keysAndValues ...interface{}) {
+	if v.enabled {
+		if v.logr != nil {
```

Comment:
This code repeats with one in `func InfoS`, it would not be good if they diverge.
Could you remove this duplication?

Reply:
done